dim-web/src/routes/user.rs

```diff
@@ -0,0 +1,296 @@
+//! This module contains all docs and APIs related to users and user metadata.
+use axum::response::{IntoResponse, Response};
+use axum::{extract, Extension};
+
+use dim_database::DbConnection;
+use dim_database::DatabaseError;
+use dim_database::asset::Asset;
+use dim_database::asset::InsertableAsset;
+use dim_database::user::User;
+
+use http::StatusCode;
+
+use serde::Deserialize;
```

Comment:
style nit: the imports here are too separated. the imports from http, serde uuid and thiserror can be put together without newlines, we will keep one newline above the http import tho.